station/utils.py

```diff
@@ -106,7 +117,12 @@ def open_config() -> Configuration:
     if not config_exists:
         directory = os.path.dirname(config_path)
         os.makedirs(directory, exist_ok=True)
-        shutil.copyfile('config.yml.template', config_path)
+
+        # Need to get the current directory, so this should work regardless if this command is issued from
+        # top dir or from station/ subdir.
+        curr_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Your comment is ambiguous. 
If you want to get current working directory you should use ```os.getcwd()```.
This line returns the path to the directory of "utils.py" file.

Reply:
Yes and that was exactly the goal. Both invocations:

```
python3 station/cli.py pass
```

and

```
cd station/
python3 cli.py pass
```

should result in the same directory being used. os.getcwd() would return different ones. I see why this may be confusing. The naming of the variable as curr_dir was unfortunate. Corrected the comment and renamed variable to avoid confusion.